app/locales/ur/overlap.json

```diff
@@ -0,0 +1,10 @@
+{
+  "overlap_title": "اوورلیپ چیک کریں",
+  "overlap_para_1": "سبز حلقے آپ کے مقامات کا گچھا ہیں ، بڑا کرنے کے لئے یہاں دبائیں۔ لال نشان آپ کے اندراج شدہ مقامات ہیں۔ \ n \ n بخوبی جامنی رنگ کے حلقوں کے بطور دکھائے جانے والے تصدیق شدہ مقدمات کی عوامی معلومات ڈاؤن لوڈ کرنے کے لئے بٹن دبائیں۔",
```

Comment:
Two points,
1. there is a space between \ and n. There shouldn't be any space in the newline character (should be \n) 
2. The value of "overlap_para_1" was updated recently. Does it correctly translate to the following?
 "overlap_para_1": "The green trail represents your location history\n\nThe light purple circles represent the public dataset"